Khe Sanh Update

Huckleberry87 opened the pull request:
Updated Khe Sanh to new standard.

Overhaul; way too many updates to list by the numbers.

Overview:
Obj 1 - Reduce overall assets.
Obj 2 - Update vehicles to spawn system.
Obj 3 - Move everyone closer together.

-Spike Teams to the north of the runway. (From west to east, Tigers, Spike Team, UDT, ANZAC)
-Air Force on the west end of the runway. (633rd and Satan's Angels)
-ACAV is west of Mike Force, and is surrounded by Marines to the south, Blackhorse to the east, and ARVN to the west. 
-7th, Muskets, MACV, and MPs is East of Mike Force. 
-Mike Force is now the place to go to load onto helos. 
-Green Hornets is where you spawn to get a helo.

Added Dock teleporter to options, UDT is up at Khe Sanh, with the dock down at the river side.


CamLaoNam -
Moved TF to former 31st MEU base and Spike Team to former Tiger Force Hangar. Base rework complete.
MEU Arsenal and Teleporter now work."
ARVN has a statics spawner.
633rd Helos are now combined into a single spawner type across the 3 pads.
Added Teleporter and destination to CamLaoNam, for continuity.

KNOWN ISSUE: Carrier Spawn from respawn is ... not working as intended on Khe Sanh

Huckleberry87 committed:
Khe Sanh Update

Updated Khe Sanh to new standard.

Overhaul; way too many updates to list.

Overview:
Spike Teams to the north of the runway.
Air Force on the west end of the runway.
ACAV is west of Mike Force, and is surrounded by Marines to the south, Blackhorse to the east, and ARVN to the west.
7th, Muskets, MACV, and MPs is East of Mike Force.
Mike Force is now the place to go to load onto helos.
Green Hornets is where you spawn to get a helo.

Added Dock teleporter to options, UDT is up at Khe Sanh, with the dock down at the river.

Added Teleporter and destination to CamLaoNam, for continuity.

KNOWN ISSUE: Carrier Spawn from Teleporter is ... not working as intended.

Huckleberry87 committed:
TF Base Rework

Moved Tiger Force to prior MEU Base. Added an extra helo.

Moved Spike Team to prior Tiger Force hangar. Added an extra helo.

Huckleberry87 committed:
Hotfix for TF Rework

Issue when Chinook spawns in. Removed obstacles.
Tweaked minor cosmetic issues that would break mY ImMeRsIoN.

Huckleberry87 committed:
MEU Hotfix and ARVN Statics

MEU - Fixed Arsenal and Teleporter issue.

ARVN - Added Statics to code and editor as requested

Huckleberry87 committed:
633rd Helo Realignment

"The request is to turn the 3 helo spawners into general spawners with 3 categories of
Heavy lift: CH-47 Cargo, CH-47 Infantry
Utility Lift: UH-1F, UH-1D, OH-6A
Medical Lift: UH-1B Med, UH-1D Med, CH-47 Med"

Huckleberry87 reviewed (commented):
Tested all locally. LGTM.

dijksterhuis commented:
CamLaoNam updates look good. 2x things on Tigers

1. Terrain hides for helos -- merge the 2x terrain hides for each helo spawn into one larger one -- remove the sandbag revetments as well
2. Airborne flags at the base? are they not meant to have the sog flag?


![20240621203038_1](https://github.com/Bro-Nation/Mike-Force/assets/11841332/18933030-d7b4-4b71-a9bc-83e4dd9d1122)

![20240621203019_1](https://github.com/Bro-Nation/Mike-Force/assets/11841332/fc1159ce-fc93-4027-938f-a0b94cbb5d16)

dijksterhuis commented:
MPs mentioned wanting some transport trucks or the MP dirt rangers too? saw something about that somewhere

dijksterhuis commented:
### khe sanh

switch these around so welcome billboard is the first one people see

![20240621203604_1](https://github.com/Bro-Nation/Mike-Force/assets/11841332/bbf493b5-aa94-4532-ba28-38e8574600b9)

flip UDT sign around so it is facing the runway (put it in near the base) --> match the other spike teams
![20240621203651_1](https://github.com/Bro-Nation/Mike-Force/assets/11841332/49995f47-1dbc-4ec6-84fc-c0809f892834)

blackhorse have two signs? leave the one facing the runway, get rid of the other?
![20240621203752_1](https://github.com/Bro-Nation/Mike-Force/assets/11841332/473046e5-9b5c-4255-b538-cd4da91de6f8)

non functional ARVN vehicle spawner
![20240621204027_1](https://github.com/Bro-Nation/Mike-Force/assets/11841332/02268a42-3315-4f05-803a-a1f773fdcb45)

feels like there's a LOT of vehicles here. we might be able to get rid of some trucks?
![20240621204102_1](https://github.com/Bro-Nation/Mike-Force/assets/11841332/2359d7b4-0e5d-4676-bcdc-0ac9b5301e7d)

no sign at USMC
![20240621204138_1](https://github.com/Bro-Nation/Mike-Force/assets/11841332/0612194b-62b5-471f-adf1-f1c627e05eff)

monties map marker
![20240621204147_1](https://github.com/Bro-Nation/Mike-Force/assets/11841332/e822dc6c-ae03-45d3-83e8-caa9291fb985)

7th cav map marker not captialised (should be a string table lookup??)
![20240621204226_1](https://github.com/Bro-Nation/Mike-Force/assets/11841332/2f07e02e-8abb-4775-b346-f7b68e8bb139)

DC jet spawners not working
![20240621204308_1](https://github.com/Bro-Nation/Mike-Force/assets/11841332/9df322aa-3f9c-40ef-954a-df981e2d2d86)

DC ground vehicles explosions -- probably need to move the apart a bit, find some flatter terrain or raise the spawn arrow up a bit

![20240621204932_1](https://github.com/Bro-Nation/Mike-Force/assets/11841332/625830a8-d531-4cb8-b12f-24ae7214afb8)

giving DC an airfield has made the map feel, dunno, like there's less up in that area. dunno how i feel about this. i quite like the last AO is the DC airfield (that doesn't have any planes, but there we are)

also -- there hasn't been DC Fixed Wing CAS on Khe Sanh before, and i'm wondering if we should keep it as the "no DC fixed wing cas" map. there's hella fewer trees on Khe Sanh, so it's prime Strella adventures.... don't really ever need CAS on that map as DC.

![20240621205054_1](https://github.com/Bro-Nation/Mike-Force/assets/11841332/866ef22a-be23-44d6-a06e-44a0ab2bc527)

dijksterhuis reviewed (commented):
added comments -- Tigers base edit and MPs truck or dirt rangers or whatever they asked for and i'm happy to merge and put on MF2 (we'll hold back with Khe Sanh as discussed)

Huckleberry87 commented:
> 1. Terrain hides for helos -- merge the 2x terrain hides for each helo spawn into one larger one -- remove the sandbag revetments as well
> 2. Airborne flags at the base? are they not meant to have the sog flag?

1. Damn... ok.

2. Tigers are a part of the 101st. 1 BN / 327 BDE / 101 DIV

dijksterhuis commented:
> 2. Tigers are a part of the 101st. 1 BN / 327 BDE / 101 DIV

errr, yes. erm. errr.

look! over there! in the opposite direction!

*runs away*

dijksterhuis commented:
> > 1. Terrain hides for helos -- merge the 2x terrain hides for each helo spawn into one larger one -- remove the sandbag revetments as well

Didn't really explain this -- gonna be hard for some of them to land on those points and might as well combine the 2x into 1x larger one.

dijksterhuis reviewed (commented):
2x more for khe sanh

boat dock arsenal / trash can not working
![20240621211442_1](https://github.com/Bro-Nation/Mike-Force/assets/11841332/e994d70f-4440-4425-9aa7-640552459d05)

rust is gonna want some UDT whitelisted boats.... I just know it 😂 

we can maybe fit in another 2x boats i think -- so could do 3x UDT and 3x public similar to cam lao
![20240621211451_1](https://github.com/Bro-Nation/Mike-Force/assets/11841332/1777f3af-4f7c-4c38-bbb1-3ea22690d9f9)

dijksterhuis commented:
comments addressed minus DC jets addition + motor pool

TODO -- trashcans!

dijksterhuis commented:
simple object picnic tables etc added -- cam lao and khe sanh

dijksterhuis commented:
Reminder -- libations issue. crates are falling through the map on cam lao nam

boxes should be raised up

khe sanh + CLN done

dijksterhuis commented:
memorials added khe sanh

dijksterhuis commented:
camlaonam hummingbird moved

dijksterhuis commented:
khe sahn zones added and moved

dijksterhuis commented:
mp troop transport vehicles

dijksterhuis commented:
welcome sign added to CLN + KS

Huckleberry87 committed:
Dev Update 21Jun

Finalize CLN
Finalize KS for first update
Added MP Trucks
Added Welcome Sign
Added/Moved KS AOs

dijksterhuis commented on mission/config/subconfigs/vehicle_respawn_info.hpp:
```diff
@@ -968,6 +1011,14 @@ class spawn_point_types {
 					"vn_b_wheeled_m151_02_mp",
 				};
 			};
+			class transport {
```
duplicate name!

dijksterhuis replied:
fixed with push

Huckleberry87 replied:
Damn! Good catch. I blame Renegade.

dijksterhuis committed:
Fixup: Duplicate spawner category name.

dijksterhuis committed:
Fixup: MP truck category icon.

dijksterhuis commented:
Also -- one broken 633rd spawn 

brkPain

![20240622051919_1](https://github.com/Bro-Nation/Mike-Force/assets/11841332/24e80538-e1e4-473b-8169-1fbcadc00a24)

dijksterhuis committed:
KheSanh: map init 633rd spawner issue fix.

dijksterhuis commented:
> Also -- one broken 633rd spawn
> 
> brkPain

fixed

dijksterhuis commented:
LGTM!
